### ISSUE TITLE
Vlastník bez názvu

### ISSUE THREAD

**Issue by BrozekAles** [NKCR-INPROVE/evidence.periodik#178]
Domnívám se, že by se neměl dostat na levé straně do "faset" Vlastník s označením "bez názvu". Buď je to nyní MZK, VKOL, NKP nebo SVKUL a nikdo další. Když se vyfiltruje vlastník "bez názvu" a rok 1976, tak v období od 1.7.1976-30.9.1976 podle čárového kódu se tam zařadily údaje ze dvou svazků MZK a jednoho svazku VKOL. Proč? 
![Snímek obrazovky 2022-02-07 v 14 26 10](https://user-images.githubusercontent.com/7532753/152796682-7d25c943-ca53-4c9a-aa2c-bc7bd9d61554.png)

**Comment by zabak** [comment]
Bude nutná oprava v datech, předpokládám. A mělo by být zajištěno, že není možné vytvořit vlastníka bez názvu, přinejmenším proto, že každý uživatel patří do nějaké knihovny a pokdu něco zadává, tak zadává to co patří dané knihovně. Což je i předmětem #158

**Comment by Kretiss** [comment]
@zabak Prosím o kontrolu, nasazeno na testu (nutnost vyplnění vlastníka).

**Comment by BrozekAles** [comment]
Ano, funguje to, díky.

**Comment by zabak** [comment]
@Kretiss prosím při releasu nezapomeňte opravit i stávající ostrá data.

**Comment by Kretiss** [comment]
@zabak Jak to myslíte? Já nevím do jakých knihoven dané svazky patří a těch svazků je takhle vytvořených celkem dost..

**Comment by BrozekAles** [comment]
Předpokládám, že svazky bez vlastníka budeme sami v knihovnách mazat, protože se pak vytvářely opravené svazky s vlastníkem, takže často bude duplicita. Je potřeba, aby ale byla funkční ikona pro mazání jednotlivých exemplářů i svazku

**Comment by zabak** [comment]
@Kretiss na základě toho co píše @BrozekAles by bylo nejlepší svazky bez vlastníka hromadně smazat.

**Comment by BrozekAles** [comment]
Ano, taky bych byl pro to, aby se svazky bez vlastníka hromadně smazaly

**Comment by korhon** [comment]
Souhlasím, není reálné to zpětně dohledávat. I když podle čárového kódu by to možná šlo

**Comment by Kretiss** [comment]
@zabak Data upravena na produkci
